### ISSUE TITLE
generate list of images required by cdk

### ISSUE THREAD

**Issue by kwmonroe** [charmed-kubernetes/jenkins#377]
We're going to host images required by CDK (heapster, dashboard, coredns, etc) in a registry. Many images used by upstream k8s are conveniently discoverable when we build cdk-addons.  There's a PR to include a new makefile target that returns a list of these images + versions:

https://github.com/charmed-kubernetes/cdk-addons/pull/95

When that functionality lands, we should update the cdk-addons build job to dump that output to a publicly accessible file that contains a list of images by release.  This file would be linked from our docs so people can find the images used in each cdk release.

**Comment by battlemidget** [comment]
Run a job to build list and commit to bundle cdk repo

**Comment by kwmonroe** [comment]
#379 is a WIP to solve this.  The idea is that a simple text file will live in the bundle repo (https://github.com/charmed-kubernetes/bundle). That file will look like:
```
v1.14-static: nexus3:latest ...
v1.14-upstream: k8s.gcr.io/addon-resizer:1.8.4 coredns/coredns:1.4.0  k8s.gcr.io/heapster-grafana-amd64:v4.4.3 ...
v1.14.1-static: nexus3:latest ...
v1.14.1-upstream: k8s.gcr.io/addon-resizer:1.9.11 coredns/coredns:1.4.0  k8s.gcr.io/heapster-grafana-amd64:v4.4.3 ...
```

The `-static` values are manually updated as needed (these aren't expected to change very often).  The `-upstream` values are generated by `make upstream-images` from the cdk-addons repo; each time cdk-addons is built for a particular release, CI will update this text file with the new image versions.

Finally, all images would be pushed to the Canonical registry.  Any glaring problems with this so far?

**Comment by battlemidget** [comment]
I think that looks ok, are the static images not part of upstream, more like one-offs for other images?

**Comment by kwmonroe** [comment]
> are the static images not part of upstream, more like one-offs for other images?

Yeah, stuff like calico, rancher, and nexus3 that aren't easy to discover when we build cdk-addons.

**Comment by battlemidget** [comment]
Can this be closed now?

**Comment by kwmonroe** [comment]
Closed by #379
